src/util/bitmap.h

```diff
@@ -164,7 +165,7 @@ class Bitmap {
 
   // Rescale image to the new dimensions.
   void Rescale(const int new_width, const int new_height,
-               const FREE_IMAGE_FILTER filter = FILTER_BILINEAR);
+               const FREE_IMAGE_FILTER filter = FILTER_CATMULLROM);
```

Comment:
Why this change? Seems unrelated to the other changes in this PR. I found bilinear to work quite well for rescaling. Note that FreeImage applies an appropriate low-pass filter depending on the scaling factor, so it's not just a pure bilinear interpolation of the pixels.

Reply:
Yes that was pulled accidentally from the diff. I will revert it to its prior setting.